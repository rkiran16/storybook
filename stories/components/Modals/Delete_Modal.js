import { html } from 'lit-html';

export const DeleteModal = () => {
  return html `
    <div class="modal fade" tabindex="-1" id="deleteModal" aria-labelledby="deleteModalLabel" aria-hidden="true">
      <div class="modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm">
        <div class="modal-content">
          <div class="modal-header">
            <h5 class="modal-title" id="deleteModalLabel">Delete 23 items</h5>
            <button type="button" class="btn btn-icon" data-bs-dismiss="modal" aria-label="Close">
            <span class="material-icons ">close</span>
        </button>
          </div>
          <div class="modal-body">
            <p class="mb-0">
              This action will remove the selected items permanantly from this list.
              Are you sure you want to delete these items ?
            </p>
          </div>
          <div class="modal-footer">
            <div class="d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start">
              <button type="button" class="btn btn-primary">Delete Items</button>  
              <button type="button" class="btn btn-tertiary" data-bs-dismiss="modal">Cancel</button>
            </div>
          </div>
        </div>
      </div>
    </div>
  `
}   