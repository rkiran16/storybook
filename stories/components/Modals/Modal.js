import { html } from "lit-html";

function insertBR() {
   return html`
      <div>
       ${Array(50).fill('').map(ele => { return  html`<br>` })}
      </div>
   `
}

export const ModalHeader = () => {
  return html`
     <div class="modal-header align-items-start">
       <div>
         <span class="material-icons text-success md-48">check_circle_outline</span>
         <h5 class="modal-title" id="exampleModalLabel">Lorem ipsum dolor</h5>
       </div>
       <button type="button" class="btn btn-icon" data-bs-dismiss="modal" aria-label="Close">
            <span class="material-icons ">close</span>
        </button>
     </div>
  `
}

export const ModalFooter = () => {
   return html`
    <div class="py-3 px-4 border-top">
        <div class="form-check">
          <input type="checkbox" class="form-check-input" id="customCheck111-33" />
          <label class="form-check-label" for="customCheck111-33">Don't show this again</label>
        </div>
    </div>
    <div class="modal-footer">
        <div class="d-grid w-100 gap-3 d-sm-flex align-items-sm-center  flex-sm-row-reverse justify-content-sm-between">
          <div class="d-grid gap-3 d-sm-flex flex-sm-row-reverse">
            <button type="button" class="btn btn-primary">Primary</button>
            <button type="button" class="btn btn-outline-secondary me-md-2">Secondary</button>
          </div>
          <button type="button" data-bs-dismiss="modal" class="btn  btn-tertiary">Teritary Button</button>
        </div>
    </div>
   `
}

export const Modal = ({size = "default"}) => {
   return html`
    <div class="modal fade" tabindex="-1" id="exampleModal" aria-labelledby="exampleModalLabel" aria-hidden="true">
        <div class=${`modal-dialog modal-dialog-centered modal-fullscreen-sm-down ${size}`}>
          <div class="modal-content">
            ${ModalHeader()}
            <div class="modal-body">
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                Donec suscipit lacinia pellentesque. Maecenas a lectus in nulla condimentum accumsan finibus lobortis enim.
                Etiam pretium efficitur dolor, sed faucibus lectus.
                Proin non euismod tellus. Nunc id commodo enim. Duis efficitur convallis efficitur. Integer porttitor.
              </p>
              <div style="height: 183px; background-color: #e2e9ea"></div>
            </div>
            ${ModalFooter()}
          </div>
        </div>
   </div>
   `
}