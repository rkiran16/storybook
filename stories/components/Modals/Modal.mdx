import { Canvas, Meta, Story } from "@storybook/addon-docs";

# MODAL

<p>
  Modal Component is built using Bootstrap’s JavaScript modal plugin to add
  dialogs to your site for lightboxes, user notifications, or completely custom
  content.
</p>

<p>
  For more Info .Please follow the Bootstrap Modal{" "}
  <a href="https://getbootstrap.com/docs/5.0/components/modal/">
    Documentation
  </a>
</p>

## Use Cases

<h5 class="mt-3">
  <strong>
    Modals focus the user’s attention exclusively on one task or piece of
    information via a window that sits on top of the page content.
  </strong>
</h5>

<ul class="list-unstyled">
  <li>
    Passive: Presents information the user needs to be aware of concerning their
    current workflow. Contains no actions for the user to take.
  </li>
  <li>
    Transactional: Requires an action to be taken in order for the modal to be
    completed and closed. Contains a cancel and primary action buttons
  </li>
  <li>
    {" "}
    Danger: A specific variant of transactional modal used for destructive or irreversible
    actions.
  </li>
  <li>
    Acknowledgement: System requires an acknowledgement of the information from
    the user. Contains only a single button, commonly “OK”.
  </li>
  <li>
    Progress: Requires several steps to be completed before it can be closed.
    Contains a cancel, previous, and next/completion buttons.
  </li>
</ul>
<div class="d-flex w-100">
  <div class="w-50">
    <p>
      <strong>Modals must always include:</strong>
    </p>
    <ul class="list-unstyled">
      <li>Close button in upper right corner.</li>
      <li>Title</li>
      <li>Background (Cardinal Black 75%)</li>
    </ul>
  </div>
  <div class="w-25">
    <p>
      <strong>DO NOT use modals for:</strong>
    </p>
    <ul>
      <li>Errors in context to an action.</li>
      <li>Feedback to an action.</li>
    </ul>
  </div>
</div>

<p>
  <strong>CTA Priorities:</strong>
</p>
For accessibility, by default, all modals must have “Cancel” represented as a tertiary
button.

Most common use cases are represented in the states of each component below. As a guideline, CTA’s should be prioritized in this order when applicable.
The number of CTA’s in a modal may flex based on the context of the content.

1. “Don’t show again” (checkbox)
2. Back (Secondary button)
3. Primary CTA
4. Cancel (Tertiary button)

<p>
  <strong>Behaviors:</strong>
</p>

- For splash pages, modals may appear on page load.
- For actions, modals may appear after clicking on a tertiary CTA or button.
- x: Clicking the close x icon in the upper right will close the modal without submitting any data and return the user to its previous context.
- Click elsewhere: Clicking outside the passive modal area will automatically close the modal.
- Esc: Press ESC on the keyboard

<p>
  <strong>Max height:</strong>
</p>

- Responsive to content and user’s screen size.
- Desktop: Follow %’s below. Mobile:
- Full screen height.

<p>
  <strong>Positioning:</strong>
</p>

- Responsive to content and user’s screen size.
- Centered vertically + horizontally.

<p>
  <strong>Mandatories:</strong>
</p>

- Headline
- Must have a close button or cancel/close tertiary cta

<p>
  <strong>Soft points:</strong>
</p>

- Icon
- Content area (can include text, images, input fields, tables, interactive calendars videos, etc.)

<Canvas>
  <Story id="components-modal--standard" name="Default Modal"></Story>
</Canvas>
