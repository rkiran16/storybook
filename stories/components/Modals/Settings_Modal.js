import { html } from 'lit-html';

export const SettingsModal = () => {
  return html `
    <div class="modal" tabindex="-1" id="settingsModal1" aria-labelledby="settingsModalLabel1" aria-hidden="true">
      <div class="modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm">
        <div class="modal-content">
          <div class="modal-header align-items-center">
            <h5 class="modal-title mb-0" id="settingsModalLabel1">Edit SMS (text) authentication</h5>
            <button type="button" class="btn btn-icon" data-bs-dismiss="modal" aria-label="Close">
              <span class="material-icons ">close</span>
            </button>
          </div>
          <div class="modal-body">
            <p class="mb-1 mt-n1">Please follow the instructions below to complete your text message authentication. </p>
            <form class="mb-3 pt-4">
              <div class="form-group mb-4">
                <input type="text" id="floatingInput" placeholder="Placeholder Text" class="form-control form-control-undefined ">
                <label for="floatingInput">Label</label>
              </div>
              <div class="form-group">
                <input type="text" id="floatingInput1" placeholder="Placeholder Text" class="form-control form-control-undefined ">
                <label for="floatingInput1">Label</label>
                <div id="emailHelp" class="form-text">Didn't receive a passcode? Resend passcode</div>
              </div>
            </form>
            <small> This mobile number will be used to restore access to your account and receive notifications. </small>
          </div>
          <div class="modal-footer">
            <div class="d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start">
              <button type="button" data-bs-target="#settingsModal2" data-bs-toggle="modal" data-bs-dismiss="modal" class="btn btn-primary">Next</button>  
              <button type="button" class="btn btn-tertiary" data-bs-dismiss="modal">Cancel</button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class="modal" tabindex="-1" id="settingsModal2" aria-labelledby="settingsModalLabel2" aria-hidden="true">
      <div class="modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm">
        <div class="modal-content">
          <div class="modal-header align-items-center">
            <h5 class="modal-title mb-0" id="settingsModalLabel2">Edit SMS (text) authentication</h5>
            <button type="button" class="btn btn-icon" data-bs-dismiss="modal" aria-label="Close">
              <span class="material-icons ">close</span>
            </button>
          </div>
          <div class="modal-body">
              <p class="mt-n1">
              For your security, please enter the one-time passcode that was sent to your device.
            </p>
            <form class="pt-2">
              <div class="form-group">
                <input type="text" id="floatingInput2" placeholder="Placeholder Text" class="form-control form-control-undefined ">
                <label for="floatingInput2">Label</label>
                <div id="emailHelp1" class="form-text">Didn't receive a passcode? Resend passcode</div>
              </div>
            </form>
          </div>
          <div class="modal-footer">
            <div class="d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start">
              <button type="button" class="btn btn-primary">Verify passcode</button>  
              <button type="button" class="btn btn-tertiary" data-bs-target="#settingsModal1" data-bs-toggle="modal"  data-bs-dismiss="modal">Back</button>
            </div>
          </div>
        </div>
      </div>
    </div>
  `
}   