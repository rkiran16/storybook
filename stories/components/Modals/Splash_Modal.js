import { html } from 'lit-html';
import { Link } from "../Link/Link";

export const SplashModal = () => {
  return html `
    <div class="modal fade" tabindex="-1" id="splashModal" aria-labelledby="splashModalLabel" aria-hidden="true">
      <div class="modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm">
        <div class="modal-content">
          <div class="modal-header align-items-start">
            <div>
              <span class="material-icons text-warning md-48">warning_amber</span>
                <h5 class="modal-title" id="exampleModalLabel">Lorem ipsum dolor</h5>
            </div>
            <button type="button" class="btn btn-icon" data-bs-dismiss="modal" aria-label="Close">
            <span class="material-icons ">close</span>
        </button>
          </div>
          <div class="modal-body">
            <p>
              The recommended browsers for Order Express are Google Chrome and Microsoft Edge.
              If you do not use recommended browsers, you may encounter issues like slowness, JavaScript errors, navigation problems, lack of security,
              1-800 errors, and limited use of the Order History page.
            </p>
            <p>
              Internet Explorer will no longer be supported in 2022.
              If you are using Internet Explorer today, please plan to transition to a supported browser as soon as possible.
              You can download Google Chrome or Microsoft Edge with the links below:
            </p>
          </div>
          <div class="modal-footer d-flex justify-content-center">
            <div class="d-grid gap-3">
                ${Link({showIcon:true})}
                ${Link({showIcon:true})}
            </div>
          </div>
        </div>
      </div>
    </div>
  `
}   