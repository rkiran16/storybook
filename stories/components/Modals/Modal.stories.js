import { html } from 'lit-html';
import { Modal } from "./Modal";
import { DeleteModal } from "./Delete_Modal";
import { SplashModal } from "./Splash_Modal";
import { SettingsModal } from "./Settings_Modal";
import docs from "./Modal.mdx";

export default {
	title: "Components/Modal",
	parameters: {
		docs: {
			page: docs
		}
	},
  argTypes: {
     size: {
			name: "Size",
      defaultValue: 'small',
      control: { type: 'select' },
      description: "Select Size",
      table: {
         type: { summary: 'select' },
         defaultValue: { summary: 'small' },
      },
      options: ['small', 'medium', 'large', 'xlarge'],
			mapping: {
				small: 'modal-sm',
				medium: 'modal-md',
				large: 'modal-lg',
				xlarge: 'modal-xl'
		 	}
		}
  }
};

const DemoTemplate = (args) => {
	 return html`
	 	<button type="button" class="btn btn-primary" data-bs-toggle="modal" data-bs-target="#exampleModal">
			 Launch Modal
 		</button>
		${Modal(args)}
	 `
}

const DeleteModalTemplate = (args) => html`
			<button type="button" class="btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#deleteModal">
					Launch Delete Modal
			</button> 
			${DeleteModal(args)}
`;

const SplashModalTemplate = args => html `
		<button type="button" class="btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#splashModal">
				Launch Splash Modal
		</button> 
		${SplashModal(args)}
`

const SettingsModalTemplate = args => html`
			<button type="button" class="btn btn-outline-secondary" data-bs-toggle="modal" data-bs-target="#settingsModal1">
				Launch Settings Modal
			</button> 
			${SettingsModal(args)}
`

export const Standard = DemoTemplate.bind({});
Standard.args = {}

export const Delete_Modal = DeleteModalTemplate.bind({});
Delete_Modal.args = {}

export const Splash_Modal = SplashModalTemplate.bind({});
Splash_Modal.args = {}

export const Settings_Modal = SettingsModalTemplate.bind({});
Settings_Modal.args = {}