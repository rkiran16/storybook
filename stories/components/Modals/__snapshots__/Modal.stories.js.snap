// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Modal Delete Modal 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
			<button type=\\"button\\" class=\\"btn btn-outline-secondary\\" data-bs-toggle=\\"modal\\" data-bs-target=\\"#deleteModal\\">
					Launch Delete Modal
			</button> 
			",
    "
",
  ],
  "values": Array [
    Object {
      "_$litType$": 1,
      "strings": Array [
        "
    <div class=\\"modal fade\\" tabindex=\\"-1\\" id=\\"deleteModal\\" aria-labelledby=\\"deleteModalLabel\\" aria-hidden=\\"true\\">
      <div class=\\"modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm\\">
        <div class=\\"modal-content\\">
          <div class=\\"modal-header\\">
            <h5 class=\\"modal-title\\" id=\\"deleteModalLabel\\">Delete 23 items</h5>
            <button type=\\"button\\" class=\\"btn btn-icon\\" data-bs-dismiss=\\"modal\\" aria-label=\\"Close\\">
            <span class=\\"material-icons \\">close</span>
        </button>
          </div>
          <div class=\\"modal-body\\">
            <p class=\\"mb-0\\">
              This action will remove the selected items permanantly from this list.
              Are you sure you want to delete these items ?
            </p>
          </div>
          <div class=\\"modal-footer\\">
            <div class=\\"d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start\\">
              <button type=\\"button\\" class=\\"btn btn-primary\\">Delete Items</button>  
              <button type=\\"button\\" class=\\"btn btn-tertiary\\" data-bs-dismiss=\\"modal\\">Cancel</button>
            </div>
          </div>
        </div>
      </div>
    </div>
  ",
      ],
      "values": Array [],
    },
  ],
}
`;

exports[`Storyshots Components/Modal Settings Modal 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
			<button type=\\"button\\" class=\\"btn btn-outline-secondary\\" data-bs-toggle=\\"modal\\" data-bs-target=\\"#settingsModal1\\">
				Launch Settings Modal
			</button> 
			",
    "
",
  ],
  "values": Array [
    Object {
      "_$litType$": 1,
      "strings": Array [
        "
    <div class=\\"modal\\" tabindex=\\"-1\\" id=\\"settingsModal1\\" aria-labelledby=\\"settingsModalLabel1\\" aria-hidden=\\"true\\">
      <div class=\\"modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm\\">
        <div class=\\"modal-content\\">
          <div class=\\"modal-header align-items-center\\">
            <h5 class=\\"modal-title mb-0\\" id=\\"settingsModalLabel1\\">Edit SMS (text) authentication</h5>
            <button type=\\"button\\" class=\\"btn btn-icon\\" data-bs-dismiss=\\"modal\\" aria-label=\\"Close\\">
              <span class=\\"material-icons \\">close</span>
            </button>
          </div>
          <div class=\\"modal-body\\">
            <p class=\\"mb-1 mt-n1\\">Please follow the instructions below to complete your text message authentication. </p>
            <form class=\\"mb-3 pt-4\\">
              <div class=\\"form-group mb-4\\">
                <input type=\\"text\\" id=\\"floatingInput\\" placeholder=\\"Placeholder Text\\" class=\\"form-control form-control-undefined \\">
                <label for=\\"floatingInput\\">Label</label>
              </div>
              <div class=\\"form-group\\">
                <input type=\\"text\\" id=\\"floatingInput1\\" placeholder=\\"Placeholder Text\\" class=\\"form-control form-control-undefined \\">
                <label for=\\"floatingInput1\\">Label</label>
                <div id=\\"emailHelp\\" class=\\"form-text\\">Didn't receive a passcode? Resend passcode</div>
              </div>
            </form>
            <small> This mobile number will be used to restore access to your account and receive notifications. </small>
          </div>
          <div class=\\"modal-footer\\">
            <div class=\\"d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start\\">
              <button type=\\"button\\" data-bs-target=\\"#settingsModal2\\" data-bs-toggle=\\"modal\\" data-bs-dismiss=\\"modal\\" class=\\"btn btn-primary\\">Next</button>  
              <button type=\\"button\\" class=\\"btn btn-tertiary\\" data-bs-dismiss=\\"modal\\">Cancel</button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div class=\\"modal\\" tabindex=\\"-1\\" id=\\"settingsModal2\\" aria-labelledby=\\"settingsModalLabel2\\" aria-hidden=\\"true\\">
      <div class=\\"modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm\\">
        <div class=\\"modal-content\\">
          <div class=\\"modal-header align-items-center\\">
            <h5 class=\\"modal-title mb-0\\" id=\\"settingsModalLabel2\\">Edit SMS (text) authentication</h5>
            <button type=\\"button\\" class=\\"btn btn-icon\\" data-bs-dismiss=\\"modal\\" aria-label=\\"Close\\">
              <span class=\\"material-icons \\">close</span>
            </button>
          </div>
          <div class=\\"modal-body\\">
              <p class=\\"mt-n1\\">
              For your security, please enter the one-time passcode that was sent to your device.
            </p>
            <form class=\\"pt-2\\">
              <div class=\\"form-group\\">
                <input type=\\"text\\" id=\\"floatingInput2\\" placeholder=\\"Placeholder Text\\" class=\\"form-control form-control-undefined \\">
                <label for=\\"floatingInput2\\">Label</label>
                <div id=\\"emailHelp1\\" class=\\"form-text\\">Didn't receive a passcode? Resend passcode</div>
              </div>
            </form>
          </div>
          <div class=\\"modal-footer\\">
            <div class=\\"d-grid w-100 d-sm-flex flex-sm-row-reverse gap-3 align-items-center  justify-content-sm-start\\">
              <button type=\\"button\\" class=\\"btn btn-primary\\">Verify passcode</button>  
              <button type=\\"button\\" class=\\"btn btn-tertiary\\" data-bs-target=\\"#settingsModal1\\" data-bs-toggle=\\"modal\\"  data-bs-dismiss=\\"modal\\">Back</button>
            </div>
          </div>
        </div>
      </div>
    </div>
  ",
      ],
      "values": Array [],
    },
  ],
}
`;

exports[`Storyshots Components/Modal Splash Modal 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
		<button type=\\"button\\" class=\\"btn btn-outline-secondary\\" data-bs-toggle=\\"modal\\" data-bs-target=\\"#splashModal\\">
				Launch Splash Modal
		</button> 
		",
    "
",
  ],
  "values": Array [
    Object {
      "_$litType$": 1,
      "strings": Array [
        "
    <div class=\\"modal fade\\" tabindex=\\"-1\\" id=\\"splashModal\\" aria-labelledby=\\"splashModalLabel\\" aria-hidden=\\"true\\">
      <div class=\\"modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm\\">
        <div class=\\"modal-content\\">
          <div class=\\"modal-header align-items-start\\">
            <div>
              <span class=\\"material-icons text-warning md-48\\">warning_amber</span>
                <h5 class=\\"modal-title\\" id=\\"exampleModalLabel\\">Lorem ipsum dolor</h5>
            </div>
            <button type=\\"button\\" class=\\"btn btn-icon\\" data-bs-dismiss=\\"modal\\" aria-label=\\"Close\\">
            <span class=\\"material-icons \\">close</span>
        </button>
          </div>
          <div class=\\"modal-body\\">
            <p>
              The recommended browsers for Order Express are Google Chrome and Microsoft Edge.
              If you do not use recommended browsers, you may encounter issues like slowness, JavaScript errors, navigation problems, lack of security,
              1-800 errors, and limited use of the Order History page.
            </p>
            <p>
              Internet Explorer will no longer be supported in 2022.
              If you are using Internet Explorer today, please plan to transition to a supported browser as soon as possible.
              You can download Google Chrome or Microsoft Edge with the links below:
            </p>
          </div>
          <div class=\\"modal-footer d-flex justify-content-center\\">
            <div class=\\"d-grid gap-3\\">
                ",
        "
                ",
        "
            </div>
          </div>
        </div>
      </div>
    </div>
  ",
      ],
      "values": Array [
        Object {
          "_$litType$": 1,
          "strings": Array [
            "
         <a href=",
            " target=",
            " class=\\"d-inline-flex align-items-center text-decoration-none\\">
            <span class=\\"text-decoration-underline\\">",
            "</span>
            <span class=\\"material-icons ms-1 mt-1 md-16\\">open_in_new</span>
         </a>",
          ],
          "values": Array [
            "http://www.cardinalhealth.com",
            "_self",
            "Link action",
          ],
        },
        Object {
          "_$litType$": 1,
          "strings": Array [
            "
         <a href=",
            " target=",
            " class=\\"d-inline-flex align-items-center text-decoration-none\\">
            <span class=\\"text-decoration-underline\\">",
            "</span>
            <span class=\\"material-icons ms-1 mt-1 md-16\\">open_in_new</span>
         </a>",
          ],
          "values": Array [
            "http://www.cardinalhealth.com",
            "_self",
            "Link action",
          ],
        },
      ],
    },
  ],
}
`;

exports[`Storyshots Components/Modal Standard 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
	 	<button type=\\"button\\" class=\\"btn btn-primary\\" data-bs-toggle=\\"modal\\" data-bs-target=\\"#exampleModal\\">
			 Launch Modal
 		</button>
		",
    "
	 ",
  ],
  "values": Array [
    Object {
      "_$litType$": 1,
      "strings": Array [
        "
    <div class=\\"modal fade\\" tabindex=\\"-1\\" id=\\"exampleModal\\" aria-labelledby=\\"exampleModalLabel\\" aria-hidden=\\"true\\">
        <div class=",
        ">
          <div class=\\"modal-content\\">
            ",
        "
            <div class=\\"modal-body\\">
              <p>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
                Donec suscipit lacinia pellentesque. Maecenas a lectus in nulla condimentum accumsan finibus lobortis enim.
                Etiam pretium efficitur dolor, sed faucibus lectus.
                Proin non euismod tellus. Nunc id commodo enim. Duis efficitur convallis efficitur. Integer porttitor.
              </p>
              <div style=\\"height: 183px; background-color: #e2e9ea\\"></div>
            </div>
            ",
        "
          </div>
        </div>
   </div>
   ",
      ],
      "values": Array [
        "modal-dialog modal-dialog-centered modal-fullscreen-sm-down modal-sm",
        Object {
          "_$litType$": 1,
          "strings": Array [
            "
     <div class=\\"modal-header align-items-start\\">
       <div>
         <span class=\\"material-icons text-success md-48\\">check_circle_outline</span>
         <h5 class=\\"modal-title\\" id=\\"exampleModalLabel\\">Lorem ipsum dolor</h5>
       </div>
       <button type=\\"button\\" class=\\"btn btn-icon\\" data-bs-dismiss=\\"modal\\" aria-label=\\"Close\\">
            <span class=\\"material-icons \\">close</span>
        </button>
     </div>
  ",
          ],
          "values": Array [],
        },
        Object {
          "_$litType$": 1,
          "strings": Array [
            "
    <div class=\\"py-3 px-4 border-top\\">
        <div class=\\"form-check\\">
          <input type=\\"checkbox\\" class=\\"form-check-input\\" id=\\"customCheck111-33\\" />
          <label class=\\"form-check-label\\" for=\\"customCheck111-33\\">Don't show this again</label>
        </div>
    </div>
    <div class=\\"modal-footer\\">
        <div class=\\"d-grid w-100 gap-3 d-sm-flex align-items-sm-center  flex-sm-row-reverse justify-content-sm-between\\">
          <div class=\\"d-grid gap-3 d-sm-flex flex-sm-row-reverse\\">
            <button type=\\"button\\" class=\\"btn btn-primary\\">Primary</button>
            <button type=\\"button\\" class=\\"btn btn-outline-secondary me-md-2\\">Secondary</button>
          </div>
          <button type=\\"button\\" data-bs-dismiss=\\"modal\\" class=\\"btn  btn-tertiary\\">Teritary Button</button>
        </div>
    </div>
   ",
          ],
          "values": Array [],
        },
      ],
    },
  ],
}
`;
